### ISSUE TITLE
BIDSLayout facade

### ISSUE THREAD

**Issue by erdalkaraca** [ANCPLabOldenburg/ancp-bids#1]
Mimic BIDSLayout of pybids project using the new implementation

**Comment by ancpJR** [comment]
This would be super useful for all the code that uses pybids layout

**Comment by tsalo** [comment]
Would this be a good place to build a list of methods/behaviors that are important for BIDSLayouts?

**Comment by ancpJR** [comment]
Yes, it would be good. We could use the google doc linked on the project page. 
When setting up a list of requirements I would suggest to keep in mind that BIDSLayout should handle modality independent and and modality specific information about datasets.

**Comment by erdalkaraca** [comment]
@tsalo I copied the contents of the hackmd to this issue, so we can better track its progress and GitHub nicely visualizes this.

> @ancpJR that Doc is a slideshow though. I threw together a HackMD, but we can move the contents wherever you prefer: https://hackmd.io/@MK6j_BTkRlehcyRZrvwQ1A/ByIp4HAMK/edit
> 
> Warning: The list of methods in the HackMD is almost certainly incomplete.

**Comment by erdalkaraca** [comment]
A minimal set of pybids functionality has been implemented. Closing for now.
If something is missing, please open a new issue.
